cumulus/parachains/runtimes/people/people-westend/src/lib.rs

```diff
@@ -136,6 +136,28 @@ pub fn native_version() -> NativeVersion {
 	NativeVersion { runtime_version: VERSION, can_author_with: Default::default() }
 }
 
+/// We allow for 2 seconds of compute with a 6 second average block.
+const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
+	WEIGHT_REF_TIME_PER_SECOND.saturating_mul(2),
+	cumulus_primitives_core::relay_chain::MAX_POV_SIZE as u64,
+);
+
+/// Maximum number of blocks simultaneously accepted by the Runtime, not yet included
+/// into the relay chain.
+const UNINCLUDED_SEGMENT_CAPACITY: u32 = 3;
+/// How many parachain blocks are processed by the relay chain per parent. Limits the
+/// number of blocks authored per slot.
+const BLOCK_PROCESSING_VELOCITY: u32 = 1;
+
+/// This determines the average expected block time that we are targeting.
+/// Blocks will be produced at a minimum duration defined by `SLOT_DURATION`.
+/// `SLOT_DURATION` is picked up by `pallet_timestamp` which is in turn picked
+/// up by `pallet_aura` to implement `fn slot_duration()`.
+///
+/// Change this to adjust the block time.
+pub const MILLISECS_PER_BLOCK: u64 = 6000;
+pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
```

Comment:
Well, I'd assume that they are parameters adjustable per parachain, and in theory, they could diverge in the future :thinking: 

Not sure if that makes sense though.

Reply:
Yes, but they were previously using the same constants imported from the same place. If there's a need to override a value for some parachain, it can simply not import it declare it's own constant.

Reply:
I was thinking the same. It would fit in well in the new parachains-constants crate, Should be straightforward to abstract this out but could also be a follow up PR.

Reply:
Fair enough! Moved them.